src/context/WalletProvider/Keplr/components/Connect.tsx

```diff
@@ -0,0 +1,87 @@
+import { Window as KeplrWindow } from '@keplr-wallet/types'
+import React, { useState } from 'react'
+import { RouteComponentProps } from 'react-router-dom'
+import { ActionTypes, WalletActions } from 'context/WalletProvider/actions'
+import { KeyManager } from 'context/WalletProvider/KeyManager'
+import { setLocalWalletTypeAndDeviceId } from 'context/WalletProvider/local-wallet'
+import { useWallet } from 'hooks/useWallet/useWallet'
+
+import { ConnectModal } from '../../components/ConnectModal'
+import { KeplrConfig } from '../config'
+
+declare global {
```

Comment:
this should probably go in a type def file outside of this component as we're extending `global` here